src/map/modifier.h

```diff
@@ -980,13 +980,19 @@ enum class Mod
     DIG_BYPASS_FATIGUE = 1074, // Chocobo digging modifier found in "Blue Race Silks". Modifier works as a direct percent.
     BREATH_DMG_DEALT   = 1075, // Breath damage dealt
 
+    // New movement speed modifiers.
```

Comment:
maybe just throw there right under where `move` is waiting to get deleted and `mount_move` sits